hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -697,12 +697,11 @@ private boolean createKey(long globalKeyNumber) {
         try (Scope writeScope = GlobalTracer.get().buildSpan("writeKeyData")
             .startActive(true)) {
           long keyWriteStart = System.nanoTime();
-          for (long nrRemaining = keySize - randomValue.length;
+          for (long nrRemaining = keySize;
                nrRemaining > 0; nrRemaining -= bufferSize) {
             int curSize = (int) Math.min(bufferSize, nrRemaining);
             os.write(keyValueBuffer, 0, curSize);
           }
-          os.write(randomValue);
```

Comment:
I think key write validation should be adjusted, too.

`keyValueBuffer` is common for all keys, while `randomValue` is unique per key.  The latter is used below in key write validatation (if requested).  So validation fails as is, since it still uses `randomValue`.

Reply:
Fixed it.

Reply:
Thanks for fixing it.  Now `randomValue` is unused, could be removed completely.  (I think findbugs complains about that, too.)

Reply:
Fixed it.

Reply:
Thank you.